examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart

```diff
@@ -34,14 +39,19 @@ abstract class ReminderManageBlocStates {
 
   ConnectableStream<Result<IdentifiablePair<ReminderModel>>> get onUpdated;
 
-  ConnectableStream<Result<ReminderModel>> get onCreated;
+  Stream<Result<ReminderModel>> get onCreated;
+
+  Stream<String> get name;
+
+  Stream<String?> get nameErrorMessage;
+
+  Stream<bool> get isNameValid;
```

Comment:
Change the type of the `onCreated` stream to `ConnectableStream`.
Remove the `nameErrorMessage` state.
Remove the `isNameValid ` state.